bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/GarbageCollectorThread.java

```diff
@@ -755,8 +759,9 @@ protected void extractMetaFromEntryLogs() throws EntryLogMetadataMapException {
                     // ledgers anymore.
                     // We can remove this entry log file now.
                     LOG.info("Deleting entryLogId {} as it has no active ledgers!", entryLogId);
-                    removeEntryLog(entryLogId);
-                    gcStats.getReclaimedSpaceViaDeletes().addCount(entryLogMeta.getTotalSize());
+                    if (removeEntryLog(entryLogId)) {
+                        gcStats.getReclaimedSpaceViaDeletes().addCount(entryLogMeta.getTotalSize());
+                    }
```

Comment:
we should increment some stats (reclaimFailedToDelete) in case of error/else block to enable monitoring of such cases

Reply:
make sense to me. I will add the stats for this.